static/js/services/http-wrapper.js

```diff
@@ -33,13 +38,13 @@ var _ = require('underscore');
     function($http, $q, ActiveRequests) {
 
       var wrap = function(args, fn) {
-        args[args.length - 1] = args[args.length - 1] || {};
-        if (args[args.length - 1].timeout === false) {
+        var options = args[args.length - 1] = args[args.length - 1] || {};
```

Comment:
But we need to pass `args` to the `apply()` method in `var promise = fn.apply(this, args);`